config/checkstyle-examples-checks.xml

```diff
@@ -89,7 +89,7 @@
     <property name="max" value="60"/>
   </module>
   <module name="LineLength">
-    <property name="max" value="85"/>
+    <property name="max" value="120"/>
```

Comment:
This value set low on purpose.
Reason is we try to be very short on width in examples to be sure that example code is easy to read.

Lets cut message to:
`// violation below 'Must include .* @deprecated Javadoc tag with description.'`

may be later on as we find more cases where `.*` is not avoidable, we will improve macros code to search replace it to `...` to more natural for humans
`// violation below 'Must include ... @deprecated Javadoc tag with description.'`

Reply:
When i use` // violation below 'Must include .* @deprecated Javadoc tag with description.'`
`Forbidden usage of regex: .* and .+ are not allowed [forbiddenRegexpInViolationMessage]`

When i use `// violation below 'Must include ... @deprecated Javadoc tag with description.'`, my test case fail citing below:
Actual and expected violations differ.
```
expected to match: 18:(?:\d+:)?\s.*Must include ... @deprecated Javadoc tag with description..*
but was          : 18: Must include both @java.lang.Deprecated annotation and @deprecated Javadoc tag with description.
```

Reply:
@romani we disallowed regexp in examples tests :)

We need to revisit criteria for examples tests comments. I am personally good to extend these to 100 characters. Another alternative is to do only `// violation` comment on same line, then have another single line comment above or below with some explanation that may not exactly match the check's violation message.

Reply:
This message needs 120

Reply:
>`When i use // violation below 'Must include ...`

Usage of ... it was my dream/idea on how we can work around in future.

To make fix easy we can update our matching to violation message with tiny code that search-replace any `...` to `.*`.

But it might collide with
`public void myMethod(String... strings)` as three dots already in use in java.
We can use 4 dots `....`.

@nrmancuso , how do you like this idea?

Reply:
@relentless-pursuit , while brainstorming happening please use violation message in comment as `@deprecated Javadoc tag with description.` it is enough. Not ideal but ok.

Reply:
done

Reply:
I keep this conversation as not resolved to let other reviewer see why we did such compromise.

Reply:
@romani let's create an issue on https://github.com/checkstyle/checkstyle/pull/13731#discussion_r1323249392 and discuss there